index.js

```diff
@@ -1,75 +1,96 @@
-/* checking the local storage for a book collection list. If non present,
-we create a new array to store new books. */
-
-let bookCollection = JSON.parse(localStorage.getItem('bookCollection')) || [];
-
-// Getting documents from html
+// eslint-disable-next-line max-classes-per-file
+class Bookdetails {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
-const bookCollectionList = document.querySelector('.book-list');
-const addBtn = document.getElementById('add-form');
-const bookTitle = document.querySelector('.titleinput');
-const bookAuthor = document.querySelector('.authorinput');
+class Collection {
+  static getData() {
+    // eslint-disable-next-line max-len
+    // const bookCollection = localStorage.getItem('bookCollection') === null ? [] : JSON.parse(localStorage.getItem('bookCollection'));
```

Comment:
- [ ] Consider removing all commented lines of code as it makes your source code duty unprofessional, I think the correct way of seeing your last changes is to use the github history commit.